src/lib.rs

```diff
@@ -1415,6 +1415,13 @@ where
     }
 }
 
+///
+/// Server Error Response
+///
+/// See `StandardErrorResponse` for an implementation adhering to the oauth standard.
+///
+pub trait ErrorResponse: DeserializeOwned + Serialize + Send + Sync + Debug + Display + 'static {}
```

Comment:
I think we can remove `Display` here as a trait bound. `StandardErrorResponse` implements it, but it's not a strict requirement that all error response types also do that. the other bounds seem reasonable, though.

nit: let's sort these alphabetically